python/chemiscope/jupyter.py

```diff
@@ -271,7 +271,13 @@ def show(
             # also adds an index property to have something to show in the info panel
             properties["index"] = {
                 "target": "structure",
-                "values": list(range(len(frames))),
+                "values": (
+                    list(range(len(frames)))
+                    if hasattr(frames, "__len__")
+                    # MDAnalysis `frames` will not be a list, and does not have
+                    # a `__len__`
+                    else list(range(len(frames.trajectory)))
+                ),
```

Comment:
This part of the code should not have to care about the exact structure provider. We should fine another way to express this, maybe creating the input first and only then adding this property?